cart.spec

```diff
@@ -14,7 +14,7 @@ BuildRequires: scons
 BuildRequires: libfabric-devel
 BuildRequires: pmix-devel
 BuildRequires: openpa-devel
-BuildRequires: mercury-devel
+BuildRequires: mercury-devel >= 1.0.1-16
```

Comment:
When I build with this PR that specifies mercury 616fee16a3c09eb93d804b627e64ea1c84c6b4d4 it results in a mercury library build that appears to have shared library versions 2.0.0. Is there a relationship between the version number specified here and the library build?

Reply:
There is no strong relationship today between mercury lib version and the version of our rpm packages. In part this is due to mercury for a while not following proper versioning and not updating the version correctly.

So version 1.0.1-16 only means rpm version in the daos-stack/mercury repository

Reply:
It's really a product of the fact that we are "between releases" upstream.  Since their latest release, 1.0.1 they have broken the ABI so they needed to bump their library version to 2.0.0.  But the latest release, upon which we patch up to the git commit that we want is still 1.0.1.  There is no 2.0.0 release (yet).  There doesn't even have to be even.  It's just (general library distribution) convention that they make their release versions the same as their library SONAME.

So while we might one day get a mercury-2.0.0.tar.gz on which to build an RPM, we're not there yet.